emgapi/migrations/0040_alter_study_is_public.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.12 on 2022-07-15 15:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('emgapi', '0039_run_ena_study_accession'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='study',
+            name='is_public',
+            field=models.IntegerField(db_column='IS_PUBLIC', default=False),
```

Comment:
I assume this is lossless i.e. Django doesn't drop and recreate the column?

Reply:
I'm going to revert this one... and will add a new field called status